oat-sa/extension-tao-itemqti
# Fix/adf 1664/solar design glitches

### Related to: https://oat-sa.atlassian.net/browse/ADF-1664

### Summary

Fix a few UI glitches in the backoffice when adding a passage to an item.

### Details

Passages must not have the full widget box frame, only a thin border.

### How to test

- check out the branch: `git checkout -t origin/fix/ADF-1664/solar-design-glitches`
- activate the feature flag: `php index.php 'oat\tao\scripts\tools\FeatureFlag\FeatureFlagTool' -s FEATURE_FLAG_SOLAR_DESIGN_ENABLED -v true`
- log in to TAO backoffice
- author an item, add a passage and look at its frame
